types/errors/callbacks.go

```diff
@@ -0,0 +1,20 @@
+package errors
+
+import (
+	"log"
+	"os"
+)
+
+var logger = log.New(os.Stderr, "defer_callback: ", log.LstdFlags|log.LUTC|log.Llongfile)
+
+// Callback is a type for a function which returns an error and is used as a callback
+type Callback = func() error
+
+// CallbackLog wraps an ErrCallback function which will log an error if it is returned
+func CallbackLog(f Callback) func() {
```

Comment:
This thing equals to writing a function to find a particular occurrence of a value in a slice of items and expose it publicly. Useless. This is Golang. Not Java.

Please remove this file.

Reply:
No, the function doesn't search for any _particular occurrence of a value in a slice of items_. 
This function is used for `defer` statement to catch and log errors.

Reply:
> the function doesn't search for

Sure. This functin though is as useful as `FindInSlice`-like function.